python/whylogs/core/constraints/factories/frequent_items.py

```diff
@@ -1,8 +1,30 @@
 from whylogs.core.configs import SummaryConfig
+from whylogs.core.relations import Require
 
 from ..metric_constraints import MetricConstraint, MetricsSelector
 
 
+def is_unique(column_name: str) -> MetricConstraint:
```

Comment:
This is based on a probabilistic sketch, we need to name this something like "is_probably_unique" and it would be better perhaps to use cardinality estimate ~= total count.

Reply:
based on previous conversations, I removed this until we get multi-metric constraints figured out